### ISSUE TITLE
initial registration

### ISSUE THREAD

**Issue by acidjazz** [mahaalo/mahalo#2]
## topics to discuss
* Make initial registration fairly simple
* Only involve requirements for a guide when chosen so
* Accounts can be both a guide and/or a trekker ?

**Comment by Bmoore106** [comment]
Yes! We had the same thought. Both a guide and a Trekker will have the basic same info, so should start with that
